packages/typespec-test/test/multiclient_modular/generated/typespec-ts/src/rest/foo/index.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as Parameters from "./parameters.js";
+import * as Responses from "./responses.js";
+import * as Client from "./clientDefinitions.js";
+import * as Models from "./models.js";
+import * as OutputModels from "./outputModels.js";
+import * as PaginateHelper from "./paginateHelper.js";
+
+export { createClient } from "./fooClient.js";
+export { Parameters, Responses, Client, Models, OutputModels, PaginateHelper };
```

Comment:
Another question about isUnexpected is that, in the foo subclient, even if we generated isUnexpected.ts file, we don't export it in the index file, and however it's being used in the api level, should we add this into the index file ? What if both bar and foo subclients have this isUnexpected ?

Reply:
I don't think so, users that import the api/ operations don't need to use isUnexpected right?

Reply:
the api folder will use it right ? does that count as internally ?

Reply:
I think in the single client case, we do export the isUnexpectedHelper to our customer.